### PR TITLE
Project setup with webpack

### PR DESCRIPTION
**Pull request by NoelLincoln**
Hello 👋

I was able to set up the to-do-list application as per the requirements:
- Set up a new project with webpack.
- Index.html file  set as a template using the HTML Webpack Plugin.
- Create an index.js file with an array of some simple to do tasks. Each task containing three keys:
description [string].
completed [bool].
index: [number]
- Wrote a function to iterate over the tasks array and populate an HTML div element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list  appears in order of the index values for each task. Which I achieved using array.sort() method.
-  CSS is loaded by Webpack Style/CSS Loader and fonts loaded with the loader.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  align-items: center;
+}
+
+.to-do-items {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 10px;
+
+  /* justify-content: space-between; */
```
Comment (rica213) [inline src/style.css:72]: - [ ] Please remove the commented-out code, it will make your codebase clean ❤️

File: server.js
Comment (rica213) [inline server.js:1]: - [ ] May I know the reason you set up the express server for this milestone? 🙏🏽

**Review by rica213 — CHANGES_REQUESTED** [review]
# 🟡 STATUS: REQUIRED CHANGES ♻️ 
Hi Noel, 
My name is Larissa. Your code reviewer 🥰

Great job!
Every requirements are implemented. Your project is well documented.
I just have one question and one little detail that I want you to address.



Happy coding!

## Highlights

Good Design ✔️
Professional Documentation ✔️
No Linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


# Cheers, and Happy coding 🖥️

## If anything is not 100% clear, kindly mention me in your comment and I will reply ASAP 🤞

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: server.js
Comment (NoelLincoln) [reply server.js:1]: I had not gotten the concept correctly. I have learned that there are three ways to set up webpack to automatically compile code. I had set up all three , while following the documentation but I had missed the point that I can use one of the three options and preferrably webpack-dev-server.
Thank you.

**Review by Owusu-Desmond — APPROVED** [review]
Hi @NoelLincoln , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Nice UI ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**